posts/2020-12-18-microprofile-long-running-actions-beta.adoc

```diff
@@ -12,19 +12,19 @@ open-graph-image: https://openliberty.io/img/twitter_card.jpg
 
 
 == Why do we need MicroProfile LRA? 
 
-An example scenario for Microprofile LRA might be a holiday booking system which has separate mircoservices to book a flight, a hotel and a taxi. To book a holiday, all three services must complete successfully. If any one service fails, all the completed steps must be rolled back. In this system, a single transaction spans multiple services and databases. This transaction is an example of a Long Running Action.
+An example scenario for Microprofile LRA might be a holiday booking system that uses three separate mircoservices to book a flight, a hotel and a taxi. To book a holiday, all three services must complete successfully. If any one service fails, all the completed steps must be rolled back. In this system, a single transaction spans multiple services and databases. This transaction is an example of a long running action.
 
-In traditional applications, ACID is the well-known transaction model and the most commonly used atomic commit protocol is the two-phase commit. In an ACID transaction, tightly-coupled XA resources can be directed to "rollback" by a coordinating transaction manager. This model introduces resource locking and global rollback, which does not scale well and therefore is not suitable for microservice architecture. As a consequence, the link:https://developer.ibm.com/depmodels/microservices/articles/use-saga-to-solve-distributed-transaction-management-problems-in-a-microservices-architecture#saga[SAGA pattern] was defined for microservice transaction architecture. A SAGA is a sequence of transactions. If one transaction fails, the saga undoes all of the preceding transactions. It aims for eventual consistence. MicroProfile LRA is an implementation of the SAGA pattern.
+In traditional applications, ACID is the well-known transaction (XA) model. In an ACID transaction, tightly-coupled XA resources can be directed to "rollback" by a coordinating transaction manager. This model introduces resource locking and global rollback, which does not scale well and therefore is not suitable for microservice architecture. As a consequence, the link:https://developer.ibm.com/depmodels/microservices/articles/use-saga-to-solve-distributed-transaction-management-problems-in-a-microservices-architecture#saga[SAGA pattern] was defined for microservice transaction architecture. A SAGA is a sequence of transactions. If one transaction fails, the saga undoes all of the preceding transactions. It aims for eventual consistence. MicroProfile LRA is an implementation of the SAGA pattern.
```

Comment:
"In traditional applications, ACID is the well-known transaction (XA) model." I don't think this sentence is correct, as XA is a commit protocol that can be used in a ACID model but it is not the whole transaction model